src/modules/airspeed_selector/AirspeedValidator.cpp

```diff
@@ -249,11 +247,7 @@ AirspeedValidator::check_airspeed_innovation(uint64_t time_now, float estimator_
 			_aspd_innov_integ_state = 0.f;
 		}
 
-		if (_tas_innov_integ_threshold > 0.f && _aspd_innov_integ_state < _tas_innov_integ_threshold) {
-			_time_last_tas_pass = time_now;
-		}
-
-		_innovations_check_failed = (time_now - _time_last_tas_pass) > TAS_INNOV_FAIL_DELAY;
+		_innovations_check_failed = _tas_innov_integ_threshold > 0.0f && _aspd_innov_integ_state > _tas_innov_integ_threshold;
```

Comment:
Do you think 
`_tas_innov_integ_threshold > 0.0f`
is even necessary?

Reply:
@sfuhrer You are right that it's not absolutely required. I was actually aware of this and still left it. But ok to remove it.